### ISSUE TITLE
Footer

### ISSUE THREAD

**Issue by Ian-Stewart-Binks** [Courseography/courseography#564]
Courseography could use a footer. Things that could go here are: the 'About' link, the 'Privacy Policy' link, a 'Get Involved' link, the disclaimer.

**Comment by david-yz-liu** [comment]
Also related, putting in a contact email address.

**Comment by Ian-Stewart-Binks** [comment]
I agree, a "last updated" time for the database would be really helpful.
